<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMainWindow</class>
 <widget class="QMainWindow" name="QMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_correction">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>수정하기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_addition_incorrectWord">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>오답 추가</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_mean">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>뜻</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>35</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>시작</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_number">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>번호</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>215</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>단어 불러오기</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>오답 불러오기</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>350</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>오늘 외울꺼</string>
    </property>
   </widget>
   <widget class="QPushButton" name="move">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>이동</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_random">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>90</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>랜덤</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_prev">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>이전</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>다음</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>삭제</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>저장하기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save_currently_state">
    <property name="geometry">
     <rect>
      <x>604</x>
      <y>300</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>이대로 오답 저장하기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_integration">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>320</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>합치기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_easyWord">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>쉬운단어 추가</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_recover">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>복구하기</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
